packages/glasscoder/PKGBUILD

```
@@ -0,0 +1,26 @@
+# Maintainer: OSAMC <https://github.com/osam-cologne/archlinux-proaudio>
+# Contributor: this.ven <misgi@ven.uber.space>
+
+pkgname=glasscoder
+pkgver=1.0.4
+pkgrel=1
+pkgdesc='Minimalist audio encoder for IceCast, Shoutcast and HLS'
+arch=(x86_64 aarch64)
+url='https://github.com/ElvishArtisan/GlassCoder'
+license=(GPL2)
+depends=(alsa-lib jack2 curl lame libfdk-aac libogg libsamplerate libsndfile libvorbis openssl opus python qt5-tools taglib twolame)
+makedepends=(docbook-xsl docbook5-xml libxslt)
```

There you go, an example of optional dependencies:
```suggestion
depends=(curl lame libfdk-aac libogg libsamplerate libsndfile libvorbis openssl opus python qt5-tools taglib twolame)
makedepends=(alsa-lib docbook-xsl docbook5-xml jack libxslt)
optdepends=('alsa-lib: for ALSA audio sources'
            'jack: for JACK audio sources')
```

---

Oh, I might be wrong here, the software doesn't seem to start at all if some dependencies are not installed ...

---

I mostly agree with that, but what's about the [hicolor-icon-theme](https://archlinux.org/packages/extra/any/hicolor-icon-theme/) packages you mentioned above?

---

Shouldn't we include that, too?